assets/xml/objects/object_fish.xml

```diff
@@ -1,162 +1,315 @@
-        <Limb Name="object_fish_Standardlimb_00296C" Type="Standard" EnumName="FISHING_FISH_LIMB_0E" Offset="0x296C" />
-        <Limb Name="object_fish_Standardlimb_002978" Type="Standard" EnumName="FISHING_FISH_LIMB_0F" Offset="0x2978" />
+
+        <!-- Fish DLs -->
+        <DList Name="gFishingFishTailFinDL" Offset="0x940" />
+        <DList Name="gFishingFishCaudalPeduncleEndDL" Offset="0x9E8" />
+        <DList Name="gFishingFishTopRearFinDL" Offset="0xB00" />
+        <DList Name="gFishingFishBottomRearFinDL" Offset="0xBA0" />
+        <DList Name="gFishingFishCaudalPeduncleDL" Offset="0xC40" />
+        <DList Name="gFishingFishTopFrontFinDL" Offset="0xE30" />
+        <DList Name="gFishingFishBottomFrontFinDL" Offset="0xED0" />
+        <DList Name="gFishingFishTorsoDL" Offset="0xF70" />
+        <DList Name="gFishingFishHeadDL" Offset="0x10A0" />
+        <DList Name="gFishingFishLeftPectoralFinDL" Offset="0x1260" />
+        <DList Name="gFishingFishRightPectoralFinDL" Offset="0x1300" />
+        <DList Name="gFishingFishJawDL" Offset="0x13A0" />
+
+        <!-- Fish Textures -->
+        <Texture Name="gFishingFishTailFinTex" OutName="fishing_fish_caudal_fin" Format="rgba16" Width="16" Height="16" Offset="0x14D0" />
+        <Texture Name="gFishingFishTopScalesTex" OutName="fishing_fish_dorsal_scales" Format="rgba16" Width="16" Height="16" Offset="0x16D0" />
+        <Texture Name="gFishingFishBottomScalesTex" OutName="fishing_fish_belly_scales" Format="rgba16" Width="16" Height="16" Offset="0x18D0" />
+        <Texture Name="gFishingFishHeadTex" OutName="fishing_fish_head" Format="rgba16" Width="32" Height="32" Offset="0x1AD0" />
+        <Texture Name="gFishingFishFinTex" OutName="fishing_fish_fin" Format="rgba16" Width="16" Height="16" Offset="0x22D0" />
+        <Texture Name="gFishingFishUnusedFinTex" OutName="fishing_fish_unused_fin" Format="rgba16" Width="16" Height="16" Offset="0x24D0" />
+        <Texture Name="gFishingFishSpinyFinTex" OutName="fishing_fish_spiny_fin" Format="rgba16" Width="16" Height="16" Offset="0x26D0" />
+
+        <!-- Fish Limbs and Skel -->
+        <Limb Name="gFishingFishHeadLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_HEAD" Offset="0x28D0" />
+        <Limb Name="gFishingFishTorsoRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TORSO_ROOT" Offset="0x28DC" />
+        <Limb Name="gFishingFishTailRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TAIL_ROOT" Offset="0x28E8" />
+        <Limb Name="gFishingFishEndRootLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_END_ROOT" Offset="0x28F4" />
+        <Limb Name="gFishingFishTailFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_TAIL_FIN" Offset="0x2900" />
+        <Limb Name="gFishingFishCaudalPeduncleEndLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE_END" Offset="0x290C" />
+        <Limb Name="gFishingFishCaudalPeduncleLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_CAUDAL_PEDUNCLE" Offset="0x2918" />
```

Comment:
For the loach, you called this the "back segment":
![image](https://github.com/zeldaret/mm/assets/12245827/73b62c6f-dc28-4ce5-bbdf-04f4fccb6a7e)

For the bass, you used a pretty technical term for this limb. What do you think about using "back segment" for this one too? Maybe then what you currently call "torso" might be "middle segment"
![image](https://github.com/zeldaret/mm/assets/12245827/4ea72895-8fff-4e80-9504-28099bad5780)

Reply:
oh yea sure ill rename that one lol its cus i first used more technical terms for everything cus i asked a friend who knows thawt kind of stuff for help but yea i figured it could be simplified a whole lot ill rename those